sycl/doc/extensions/supported/sycl_ext_oneapi_prod.asciidoc

```diff
@@ -20,7 +20,7 @@
 == Notice
 
 [%hardbreaks]
-Copyright (C) 2022 Intel Corporation.  All rights reserved.
+Copyright (C) 2022-2024 Intel Corporation.  All rights reserved.
```

Comment:
This looks like a merge mistake.  PR #13623 changed the copyright date to a single year.